json.h

```diff
@@ -0,0 +1,75 @@
+#ifndef json_h
+#define json_h
```

Comment:
I think it should be kept consistent with the other header files in this project, which means using lower-cases following the double underlines.